medusajs/medusa
# fix(admin-ui): password reset flip rendering condition

**What** - Admin UI password reset page
**Why** - The rendering ternary is wrong - currently it renders the reset form when there is no token and renders the invalid token state when a token was parsed successfully
**How** - I flipped the condition
**Testing** 

- go to admin page https://xxx.app/app/reset-password
- it should render the reset link invalid state
- go to https://xxx.app/app/reset-password?token=[someValidJWTToken]
- it should render the password reset form
